### PR TITLE
[FIX] hw_drivers: add cron for HTTPS certificate update

### PR DESCRIPTION
**Pull request by lse-odoo**
Context:
When an HTTPS certificate is delivered, it is valid for ~1 month. To avoid missing the certificate, a script is automatically ran daily to check if a new HTTPS certificate is necessary.

On the IoT box, it is added in the native Unix system with this file: https://github.com/odoo/odoo/blob/16.0/addons/point_of_sale/tools/posbox/overwrite_after_init/etc/cron.daily/odoo

However, prior to this commit, there is nothing equivalent for windows

Note: the HTTPS certificate check is also done automatically when accessing the homepage.

Before this commit:
After an HTTPS certificate delivery, if we let the IoT server running non-stop (and without accessing the homepage). The HTTPS will expire without any automatic renew.

After this commit:
A "Cron Handler" have been introduced in order to manage time related actions.
For now, only some actions that have to be executed daily is implemented.

Other note:
- Using native Windows "Scheduled Task" have been proposed at: https://github.com/odoo/odoo/pull/144584
But, was judged too risky from a security point of view
- `sched` library have been chosen as it is native in python (since 2.6). Other non-native libraries might be better, but should be considered in master for stability
- `sched` default way of working is using `time.sleep`, so we basically wait for 24 hours in the handler thread. If it is an issue, we might be able to have something similar using thread.Timer:
https://docs.python.org/3/library/threading.html#timer-objects but I think it will consume more resources for no real benefit: https://stackoverflow.com/questions/13261593/resource-usage-of-time-sleep-in-loop-vs-threading-timer

opw-3617687

### CODE REVIEW

File: addons/hw_drivers/main.py [hunk 1]
```diff
@@ -13,6 +13,13 @@
 
 _logger = logging.getLogger(__name__)
 
+if platform.system() == 'Windows':
+    try:
+        import schedule
+    except ImportError:
+        schedule = None
+        _logger.error('Could not import library schedule')
```
Comment (qle-odoo) [inline addons/hw_drivers/main.py:21]: Change to _logger.warning basically it is not an error

**Comment by qle-odoo** [comment]
@robodoo delegate+

File: addons/hw_drivers/main.py (hunk 1 above)
Comment (lse-odoo) [reply addons/hw_drivers/main.py:21]: why wouldn't it be ? 5 lines below for the import of `DBusGMainLoop` it is also logged as an error. DO you want me to change this line too ?

**Comment by lse-odoo** [comment]
@robodoo r+

**Pull request closed without merging**